etcdutl/etcdutl/backup_command.go

```diff
@@ -72,7 +72,7 @@ func NewBackupCommand() *cobra.Command {
 }
 
 func doBackup(cmd *cobra.Command, args []string) {
-	HandleBackup(withV3, dataDir, backupDir, walDir, backupWalDir)
```

Comment:
Can you remove `withV3` flag?

Reply:
I mean variable `withV3` on line 45. Unfortunately GitHub doesn't allow me to comment there.